app/src/androidTest/java/ch/epfl/sweng/swenggolf/CreateOfferActivityTest.java

```diff
@@ -148,7 +148,7 @@ public void modifyingOfferViaShowOfferWorksOnlyOnCreator() {
         onView(withId(R.id.button_modify_offer)).check(matches(not(isClickable())));
     }
 
-    @Test
+    /*@Test
```

Comment:
I think it would be better to remove the defective test instead of keeping it commented out.
If we want to, we can always add it back later on and recover the code from previous commits.